src/main/java/org/radarcns/connect/mongodb/MongoDbSinkTask.java

```diff
@@ -131,6 +131,10 @@ public void put(Collection<SinkRecord> sinkRecords) {
             buffer.add(record);
             monitor.increment();
 
+            //TODO Remove this
+            log.info("{} --> {}", partition, record.kafkaOffset());
+            log.info("Buffer size now --> {}", buffer.size());
```

Comment:
Move this to the debug statement below. Perhaps the log4j config can be added to the class path in the docker container as well? For easier logging modification. Or use the log4j env configuration part of the Kafka-connect-base. See radar docker for the relevant env variable name.

Reply:
If we specify `CONNECT_LOG4J_ROOT_LOGLEVEL=DEBUG` in the ENV for the connector, then it should work. No need to include a log4j config i guess ?

Reply:
Exactly. The code just needs to be moved (with debug) two lines lower.

Reply:
done. 👍